### ISSUE TITLE
colorscheme default not properly overridden for lualine

### ISSUE THREAD

**Issue by bob3000** [LunarVim/LunarVim#3137]
### Problem description

The lualine colors are currently inconsistent with the selected colorscheme in the fields branch and line number (see screenshots).
I couldn't exactly figure out what's going on but I made some observations that might be helpful.

1. In https://github.com/LunarVim/LunarVim/blob/rolling/lua/lvim/core/lualine/components.lua#L27 I wrote `print(lvim.colorscheme)` into the conditional block and `:messages` actually printed `tokyonight` even though in my configuration I set `lvim.colorscheme` to `gruvbox-material`. When I comment out the entire if block the problem goes away.

2. In https://github.com/LunarVim/LunarVim/blob/rolling/lua/lvim/config/defaults.lua#L3 when I set the colorscheme to `gruvbox-material` it also makes the problem go away.

In summary it looks like the for some reason the default is not properly overridden by the user config but even more interestingly it only has a negative effect on the two fields mentioned above.

### LunarVim version

rolling-35cea14:q

### Neovim version (>= 0.7.2)

NVIM v0.8.0

### Operating system/version

macOS 12.6

### support info

.

### Screenshots

wrong lualine style (background is black)
![lualine-wrong](https://user-images.githubusercontent.com/1550574/193647208-c08411b9-d4eb-47ea-a242-e6679233cd7c.png)

correct lualine style (background is dark brown)
![lualine-right](https://user-images.githubusercontent.com/1550574/193647115-9f3d1ead-2205-47be-8d7d-bc21daa8653b.png)
